### PR TITLE
[rustfmt] merge imports

### PR DESCRIPTION
**Pull request by huitseeker**
This rustfmt option is only in nightly for now, so warnings may ensue. This is mostly to see if it breaks CI as currently set up.

@zekun000 @davidiw may be interested / reviewing.

### CODE REVIEW

**Comment by zekun000** [comment]
are you reading my post lol? I literally just posted about using this nightly feature, but I think we have to add nightly toolchain to CI before adding it to `rustfmt.toml`

File: language/compiler/ir-to-bytecode/syntax/src/syntax.rs [hunk 1]
```diff
@@ -3,12 +3,10 @@
 
 use anyhow::{Context, Error};
 use codespan::{ByteIndex, Span};
-use std::fmt;
-use std::str::FromStr;
+use std::{fmt, str::FromStr};
 
 use crate::lexer::*;
-use libra_types::identifier::Identifier;
```
Comment (davidiw) [inline language/compiler/ir-to-bytecode/syntax/src/syntax.rs]: rustfmt cannot fix these though :(

**Comment by dariorussi** [comment]
thanks for all the conflicts, much appreciated

**Pull request closed without merging**